backend/app/modules/user/tos/core.py

```diff
@@ -19,6 +19,12 @@ def build_consent_message(user_address: str) -> str:
     )
 
 
+def build_consent_message_patron(user_address: str) -> str:
+    return "\n".join(
+        (f"Signing this message will enable patron mode for address {user_address}")
+    )
+
+
```

Comment:
Please use this one instead: https://github.com/golemfoundation/octant/blob/develop/backend/app/legacy/crypto/eth_sign/patron_mode.py#L4

Reply:
This will also likely fix the error.